api/src/main/java/com/datastrato/gravitino/rel/expressions/Literal.java

```diff
@@ -28,7 +28,10 @@
  * Represents a constant literal value in the public expression API.
  *
  * @param <T> the JVM type of value held by the literal
+ * @deprecated This interface will be removed. Use {@link
+ *     com.datastrato.gravitino.rel.expressions.literals.Literal} instead.
  */
+@Deprecated
```

Comment:
Why do you deprecate the class, not just the method?

Reply:
In fact, I have moved this class from package `com.datastrato.gravitino.rel.expressions` to `com.datastrato.gravitino.rel.expressions.literals`, and I want users to use the new class.